homeassistant/loader.py

```diff
@@ -142,6 +142,63 @@ async def async_get_config_flows(hass: "HomeAssistant") -> Set[str]:
     return flows
 
 
+async def async_get_zeroconf(hass: "HomeAssistant") -> Dict[str, List]:
+    """Return cached list of zeroconf types."""
+    # pylint: disable=import-outside-toplevel
+    from homeassistant.generated.zeroconf import ZEROCONF
```

Comment:
Why import inside the function. I don't see a circular reference?

Reply:
I don't think its needed.  Its copied from
```
async def async_get_config_flows(hass: "HomeAssistant") -> Set[str]:
    """Return cached list of config flows."""
    # pylint: disable=import-outside-toplevel
    from homeassistant.generated.config_flows import FLOWS
```


removed